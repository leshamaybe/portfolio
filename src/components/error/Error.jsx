import React from 'react';
import Link from 'next/link';
import s from './error.module.scss'

const Error404 = () => {
    return (
        <div className={s.error}>
            <h1>404</h1>
            <h2>Ooops... There's nothing in here</h2>
            <Link href="/">&#8592;Back home page</Link>
        </div>
    );
};

export default Error404;
