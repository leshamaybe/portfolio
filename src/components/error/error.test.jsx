import Error from './Error';
import { render } from '@testing-library/react';

describe('Error', () => {
    it('Error renders correctly', async () => {
        const { container } = render(<Error />);
        expect(container).toMatchSnapshot();
    });
});
