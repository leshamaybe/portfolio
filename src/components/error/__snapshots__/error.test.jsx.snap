// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Error renders correctly 1`] = `
<div>
  <div
    class="error"
  >
    <h1>
      404
    </h1>
    <h2>
      Ooops... There's nothing in here
    </h2>
    <a
      href="/"
    >
      ‚ÜêBack home page
    </a>
  </div>
</div>
`;
